pkg/controller/storagecluster/generate.go

```diff
@@ -6,8 +6,19 @@ import (
 	ocsv1 "github.com/openshift/ocs-operator/pkg/apis/ocs/v1"
 )
 
+func generateNameForExternalCephCluster(initData *ocsv1.StorageCluster) string {
+	return generateNameForCephClusterFromString(fmt.Sprintf("%s-external", initData.Name))
+}
+
 func generateNameForCephCluster(initData *ocsv1.StorageCluster) string {
-	return generateNameForCephClusterFromString(initData.Name)
+	clusterName := ""
```

Comment:
Why would we need to change the name of the cluster? IIRC OCS-Op only supports a single CephCluster anyways.

Reply:
Idea was to make cluster identifiable by name, no other reason.